.gitignore

```diff
@@ -36,3 +36,4 @@
 /wp-content/themes/pub/wporg-learn-2020/css/print.css
 /wp-content/themes/pub/wporg-learn-2020/css/style-rtl.css
 /wp-content/themes/pub/wporg-learn-2020/css/style.css
+package-lock.json
```

Comment:
@unprintedch could I check if this change was intentional, it's generally not something that gets added to the .gitignore file.

Reply:
It is fixed.